### ISSUE TITLE
Scalability Z-Axis

### ISSUE THREAD

**Issue by herrBez** [herrBez/ethereum-scalability#12]

**Comment by herrBez** [comment]
Maybe after the description we could write that nowadays there is no partition and then we can describe the proposal of sharding. WIthout tests because they do not have implemented it. @gfornari

**Comment by herrBez** [comment]
@gfornari

**Comment by gfornari** [comment]
I'd keep it with low priority and discuss it only after a first version of the second part.

**Comment by herrBez** [comment]
Ok, but what would you write about Z-axis?

**Comment by herrBez** [comment]
@gfornari
